thunder/executors/torch_autograd.py

```diff
@@ -444,6 +444,12 @@ def split_forward_backward(computation_trc: TraceCtx, compile_data, compile_stat
     # We only want the forward function to be called with `te.fp8_autocast` manager.
     bw_extrace._include_te_fp8_autocast = False
 
+    # Should this be a post optmization transform?
+    # We only want to apply it on backward trace.
```

Comment:
It's okay to do this during splitting when the backward execution trace is created.